Apps/WebClient/src/ClientApp/src/components/datePicker.vue

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+import { BFormDatepicker } from "bootstrap-vue";
+import Vue from "vue";
+import {
+    Component,
+    Emit,
+    Model,
+    Prop,
+    Ref,
+    Watch,
+} from "vue-property-decorator";
+import { minLength } from "vuelidate/lib/validators";
+import { Validation } from "vuelidate/vuelidate";
+import { DateWrapper } from "@/models/dateWrapper";
+
+@Component
+export default class DatePickerComponent extends Vue {
+    @Model("change", { type: String }) public model!: string;
+    @Prop() state?: boolean;
+    @Ref("datePicker") datePicker!: BFormDatepicker;
+
+    private inputValue = "";
+
+    private mounted() {
+        this.inputValue = this.model;
+    }
+
+    private onFocus() {
```

Comment:
?

Reply:
I commented the hack to display the calendar on focus since its glitchy, we might want to look into to this later.